goth/configuration.py

```diff
@@ -292,6 +299,12 @@ def load_yaml(
             name = node["name"]
             type_name = node["type"]
             use_proxy = node.get("use-proxy", False)
+
+            payment_config_name = node.get(
+                "payment_config", DEFAULT_PAYMENT_CONFIG_NAME
```

Comment:
Apologies for the confusion, but I think for the sake of consistency with other `goth-config.yml` fields the name of the field itself should be `payment-config` (hyphen instead of underscore).

Reply:
Good find. I should have thought about it. 
cbb2254